cfgov/jinja2/v1/_includes/organisms/search-bar.html

```diff
@@ -36,33 +36,27 @@
                 block__bg-extend-{{ value.position }}
               {% endif %}
               {{ value.additional_classes }}" >
-    {% if position == 'full' %}
-    <div class="content_wrapper">
-    {% endif %}
-        <form method="get" action="{{ value.action }}" class="content-l">
-            {% if value.label %}
-                <div class="o-search-bar_label o-search-bar_col">
-                    <h4 class="u-mb0">
-                        {{ value.label | safe }}
-                    </h4>
+    <form method="get" action="{{ value.action }}" class="content-l">
+        {% if value.label %}
+        <div class="o-search-bar_label o-search-bar_col">
+            <h4 class="u-mb0">
+                {{ value.label | safe }}
+            </h4>
+        </div>
+        {% endif %}
+        <div class="o-search-bar_input o-search-bar_col">
+            <div class="input-with-btn input-with-btn__bar">
+                <div class="input-with-btn_input">
+                    <input type="text" name="q" id="q" >
                 </div>
-            {% endif %}
-            <div class="o-search-bar_input o-search-bar_col">
-                <div class="input-with-btn input-with-btn__bar">
-                    <div class="input-with-btn_input">
-                        <input type="text" name="q" id="q" >
-                    </div>
-                    <div class="input-with-btn_btn">
-                        <button class="btn" name="btnSearch" id="btnSearch">
-                            Search
-                        </button>
-                    </div>
+                <div class="input-with-btn_btn">
+                    <button class="btn" name="btnSearch" id="btnSearch">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2932/commits/6b97bcdb46bda7798c11322457401dbfb191dd37